sedai/assets/dashboards/sedai_overview.json

```diff
@@ -0,0 +1,614 @@
+{
+    "title": "Sedai Dashboard",
+    "description": "Sedai is an autonomous, agentless, and configuration-free SaaS that intelligently manages your cloud environments to prevent availability issues and optimize performance and cost — all in production. Sedai's Continuous Autonomous Availability Management (CAAM) platform helps you take full advantage of your microservices with smart signals that prevent downtime, avoid potential SLO breaches, improve latency, analyze code releases, and lower cloud costs\n\nSedai User Guide: https://sedai.gitbook.io/sedai/",
```

Comment:
```suggestion
    "description": "Sedai is an autonomous, agentless, and configuration-free SaaS that intelligently manages your cloud environments to prevent availability issues and optimize performance and cost—all in production. Sedai's Continuous Autonomous Availability Management (CAAM) platform helps you take full advantage of your microservices with smart signals that prevent downtime, avoid potential SLO breaches, improve latency, analyze code releases, and lower cloud costs. \n\nSedai User Guide: https://sedai.gitbook.io/sedai/",
```

Reply:
@cswatt, 
could you please review the removal of space before and after the '-' in the line 'and cost - all in..' ..?
it's not intended be read as a single word - but to emphasize with a pause.

Reply:
@cswatt,
I have updated the content after reviewing with our content team - the Em dash is replaced with a comma.